Assets/Scripts/CreatureStats.cs

```diff
@@ -133,19 +140,70 @@ private void DisplayPopup(string text)
         textLabelText = text;
     }
 
+    // Returns true if floats are within 15.0f of each other.
+    // compare Mathf.Approximately.
+    private bool VeryApproximately(float f1, float f2)
+    {
+        if (Mathf.Abs(f1 - f2) < 15.0f) return true;
+        return false;
+    }
+
+    // Returns true if the target is being attacked from the side or rear.
+    // Side/read attacks double the chance of a critical hit.
+    public bool IsFlanking(CreatureStats target)
+    {
+        float rotation1 = target.transform.eulerAngles.y;
+        float rotation2 = transform.eulerAngles.y;
+        bool flanking = !VeryApproximately(Mathf.Abs(rotation1 - rotation2), 180.0f);
```

Comment:
Can we simplify this by writing it like so:
`return VeryApproximately(Mathf.Abs(rotation1 - rotation2), 0f);`

Reply:
Good idea. Done.